lib/linguist/languages.yml

```diff
@@ -1508,7 +1511,9 @@ GLSL:
   - ".fshader"
   - ".geo"
   - ".geom"
+  - ".glslf"
```

Comment:
I checked with Harvester and the `extension:glslf version OR void OR layout` search query that there are enough `.glslf` files to warrant support in Linguist. I found 568 repositories by 485 users.